### PR TITLE
adding a default job with some params and example of additional job.

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

Small changes to add a job and some params via the `example.conf` file to the example collector. Doing this so it can be referred to in a guide being created for how to create a custom collector in python. 

##### Component Name

collectors/python.d.plugin/example

##### Test Plan

Tested locally on development machine.

### CODE REVIEW

**Comment by andrewm4894** [comment]
@ilyam8 this is just a small little addition to help with the Python collector guide [here](https://github.com/netdata/netdata/pull/10710/files#r595126966) e.g. so the example collector can be referenced as an example of how to use jobs with params.

**Review by ilyam8 — DISMISSED** [review]

**Comment by andrewm4894** [comment]
apologies had to make one last change to make sure "Four Lines" actually gives 4 lines so as not to confuse anyone. Good to go now.

**Review by ilyam8 — APPROVED** [review]

File: collectors/python.d.plugin/example/example.conf [hunk 1]
```diff
@@ -66,3 +66,26 @@
 # ----------------------------------------------------------------------
 # AUTO-DETECTION JOBS
 # only one of them will run (they have the same name)
```
Comment (vlvkobal) [inline collectors/python.d.plugin/example/example.conf:68]: This line looks quite strange in the new context.

File: collectors/python.d.plugin/example/example.conf [hunk 2]
```diff
@@ -66,3 +66,26 @@
...
+four_lines:
+    name: "Four Lines"      # the JOB's name as it will appear at the
+    update_every: 1         # the JOB's data collection frequency
+    priority: 60000         # the JOB's order on the dashboard
+    penalty: yes            # the JOB's penalty
+    autodetection_retry: 0  # the JOB's re-check interval in seconds
+    # some collector specific params
+    num_lines: 4            # the number of lines to create
+    lower: 0                # the lower bound of numbers to randomly sample from
+    upper: 100              # the upper bound of numbers to randomly sample from
```
Comment (vlvkobal) [inline collectors/python.d.plugin/example/example.conf:79]: I think you should add new parameter descriptions after L60 and remove `# some collector specific params`.

File: collectors/python.d.plugin/example/example.conf [hunk 3]
```diff
@@ -66,3 +66,26 @@
...
+# you would just uncomment the job configuration below.
+# two_lines:
+#     name: "Two Lines"      # the JOB's name as it will appear at the
+#     update_every: 1         # the JOB's data collection frequency
+#     priority: 60000         # the JOB's order on the dashboard
+#     penalty: yes            # the JOB's penalty
+#     autodetection_retry: 0  # the JOB's re-check interval in seconds
+#     num_lines: 2            # the number of lines to create
+#     lower: 50                # the lower bound of numbers to randomly sample from
+#     upper: 75              # the upper bound of numbers to randomly sample from
```
Comment (vlvkobal) [inline collectors/python.d.plugin/example/example.conf:91]: Please align the descriptions.

**Review by vlvkobal — CHANGES_REQUESTED** [review]

**Comment by andrewm4894** [comment]
@vlvkobal  i see what you mean - made those few changes to clean up the conf a little.

**Review by vlvkobal — APPROVED** [review]

File: collectors/python.d.plugin/example/example.conf [hunk 4]
```diff
@@ -61,8 +61,27 @@
...
+
+four_lines:
+    name: "Four Lines"      # the JOB's name as it will appear at the
+    update_every: 1         # the JOB's data collection frequency
+    priority: 60000         # the JOB's order on the dashboard
+    penalty: yes            # the JOB's penalty
+    autodetection_retry: 0  # the JOB's re-check interval in seconds
+    num_lines: 4            # the number of lines to create
+    lower: 0                # the lower bound of numbers to randomly sample from
+    upper: 100              # the upper bound of numbers to randomly sample from
```
Comment (ilyam8) [inline collectors/python.d.plugin/example/example.conf:79]: Why do you think we need to copy comments?


https://github.com/andrewm4894/netdata/blob/8482d276e2eb6f2b8736d5c560c5c126abc4f801/collectors/python.d.plugin/example/example.conf#L53-L79

Comment (andrewm4894) [reply collectors/python.d.plugin/example/example.conf:79]: to be as clear as possible for the user. e.g. most will probably just start by scanning the actual lines that are not comments in the file and then see what they are as opposed to reading the green wall of text that is all the stuff above. 

i scan the conf with my eyes - see what lines are not comments, see what they do and then got 90% of what i need to be productive. Then as needed at a later stage i may go back and read every single line of the conf comments and all.

File: collectors/python.d.plugin/example/example.conf [hunk 5]
```diff
@@ -61,8 +61,27 @@
...
+#     num_lines: 4            # the number of lines to create
+#     lower: 0                # the lower bound of numbers to randomly sample from
+#     upper: 100              # the upper bound of numbers to randomly sample from
 #
 # ----------------------------------------------------------------------
 # AUTO-DETECTION JOBS
-# only one of them will run (they have the same name)
+
+four_lines:
+    name: "Four Lines"      # the JOB's name as it will appear at the
```
Comment (ilyam8) [inline collectors/python.d.plugin/example/example.conf:72]: > at the

where? 🤔

Comment (andrewm4894) [reply collectors/python.d.plugin/example/example.conf]: fixed

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**